src/Files.App/ResourceDictionaries/PathIcons.xaml

```diff
@@ -1448,7 +1448,7 @@
 				</Setter>
 			</Style>
 
-			<Style x:Key="ColorIconPinToSidebar" TargetType="local:OpacityIcon">
+			<Style x:Key="ColorIcon.PinToSidebar.16x16" TargetType="local:OpacityIcon">
```

Comment:
I didn't realize this was also used for "Pin to start". Maybe it should just be "Pin" and "Unpin".